README.md

```diff
@@ -1190,7 +1190,26 @@ try {
 } catch (DescopeException de) {
     // Handle the error
 }
+```
+Generate a JWT for a user, simulating a signin request.
 
+```java
+JwtService jwts = descopeClient.getManagementServices().getJwtService();
+AuthenticationInfo res = jwts.signIn("dummy");
```

Comment:
I think that there is a second parameter that you must provide, no? For all of them.
```
  AuthenticationInfo signUp(String loginId, MgmtSignUpUser signUpUserDetails)
          throws DescopeException;

  AuthenticationInfo signUpOrIn(String loginId, MgmtSignUpUser signUpUserDetails)
          throws DescopeException;

  AuthenticationInfo signIn(String loginId, LoginOptions loginOptions) throws DescopeException;
```

Reply:
```
public class MgmtSignUpUser {
  private User user;
  boolean verifiedEmail;
  boolean verifiedPhone;
  String ssoAppId;
  Map<String, Object> customClaims;
}
```